esp/GBPlay/main/hardware/wifi.c

```diff
@@ -101,6 +111,14 @@ static void _on_ip_event(void* arg, esp_event_base_t event_base, int32_t event_i
 
 void wifi_initialize()
 {
+    // Load previously saved credentials
+    void* saved_networks = storage_get_blob(WIFI_SAVED_NETWORKS_STORAGE_KEY);
+    if (saved_networks)
+    {
+        memcpy(&s_saved_networks, saved_networks, sizeof(s_saved_networks));
```

Comment:
Cached in memory so we don't have to go to flash every time. I flush it to flash only when modified (network saved/forgotten due to user changing it).